plugins/labhub.py

```diff
@@ -302,11 +302,27 @@ def difficulty_level(user, iss):
                 elif self.GH3_ORG.is_member(user):
                     return True
 
+            @register_check
+            def block_gci_issue_assignment(user, iss):
+                """
+                True if the issue is not labelled with 'intiatives/gci'.
+                False if the issue has been labelled with 'initiatives.gci'.
+                """
+                if 'initiatives/gci' in iss.labels:
+                    return False
+                else:
+                    return True
+            
```

Comment:
The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `all.autopep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpruo6ink7/plugins/labhub.py
+++ b/tmp/tmpruo6ink7/plugins/labhub.py
@@ -312,7 +312,7 @@
                     return False
                 else:
                     return True
-            
+
         def eligible(user, iss):
             """
             True if all checks are passed.
```